SU-SWS/decanter
# 188 Button Tweak

# READY FOR REVIEW

# Summary
- Add more padding to the bottom on button for better optical alignment

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Run `grunt styleguide`
3. Ensure that all three button types have slightly more padding below than above their content

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- #188 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
